src/__test__/mockapi/widgets_get.json

```diff
@@ -0,0 +1,39 @@
+[
```

Comment:
This should probably be revisited since Widget objects have more properties than this mock would suggest.